src/CMakeLists.txt

```diff
@@ -69,6 +69,58 @@ quick_lint_js_add_library(
   vector.cpp
   vim-qflist-json-error-reporter.cpp
   wasm-demo-error-reporter.cpp
+  quick-lint-js/assert.h
```

Comment:
Nit: I'd like this list to be sorted lexicographically, such that `quick-lint-js/assert.h` appears between `parse.cpp` and `text-error-reporter.cpp`.